Describe: v2.6.37-rc1-1484-gc072240
commit c07224005dd3fe746246acadc9be652a588a4d7f
Author: Jiri Pirko <jpirko@redhat.com>
Date:   Thu Dec 9 03:40:30 2010 +0000

    net/ipv6/udp.c: fix typo in flush_stack()
    
    skb1 should be passed as parameter to sk_rcvqueues_full() here.
    
    Signed-off-by: Jiri Pirko <jpirko@redhat.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

diff --git a/net/ipv6/udp.c b/net/ipv6/udp.c
index 7aad127..26a8da3 100644
--- a/net/ipv6/udp.c
+++ b/net/ipv6/udp.c
@@ -602,7 +602,7 @@ static void flush_stack(struct sock **stack, unsigned int count,
 
 		sk = stack[i];
 		if (skb1) {
-			if (sk_rcvqueues_full(sk, skb)) {
+			if (sk_rcvqueues_full(sk, skb1)) {
 				kfree_skb(skb1);
 				goto drop;
 			}
